ExtraEnemyCustomization/Customizations/EnemyAbilities/Abilities/EMP/Handlers/EMPPlayerHudHandler.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+using LevelGeneration;
+
+namespace EECustom.Customizations.EnemyAbilities.Abilities.EMP.Handlers
+{
+    public class EMPPlayerHudHandler : EMPHandlerBase
+    {
+        readonly List<RectTransformComp> _targets = new();
```

Comment:
Protection definition is missing, inconsistent with rest of codebase.

```suggestion
        private readonly List<RectTransformComp> _targets = new();
```